shared/actions/chat/inbox.js

```diff
@@ -724,35 +728,26 @@ function* _inboxSynced(action: ChatGen.InboxSyncedPayload): Saga.SagaGenerator<a
   const conversation = Constants.getSelectedConversationStates(state)
   if (conversation) {
     const inbox = Constants.getInbox(state, selectedConversation)
+    const lastMessageIDWeAreShowing = Constants.lastMessageID(state, selectedConversation)
+
+    // Check the data we are given first, then our state
+    const knownMaxMessageID: ?number = latestMaxMsgID[selectedConversation] || (inbox ? inbox.maxMsgID : null)
```

Comment:
The inbox state is old, let's use the latest information we have in the action and fall back to the inbox state